build_variables.bzl

```diff
@@ -624,7 +628,7 @@ libtorch_lite_cmake_sources = sorted(
     torch_mobile_core,
 )
 
-libtorch_cmake_sources = libtorch_core_sources + libtorch_core_jit_sources + libtorch_nativert_sources
+libtorch_cmake_sources = libtorch_core_sources + libtorch_core_jit_sources + libtorch_nativert_sources + libtorch_nativert_standalone_headers
```

Comment:
What does adding headers to this do? These are sources to be compiled?

Reply:
@albanD I don't think these are source files.

Reply:
These are all `.cpp` files is what I meant.